www/redland-bindings/Portfile

```diff
@@ -35,35 +33,45 @@ pre-configure {
     }
 }
 
-variant perl5 {
-	depends_lib-append path:bin/perl:perl5
-	configure.args-append --with-perl
-	configure.perl ${prefix}/bin/perl
+if {![variant_isset perl5] && ![variant_isset php5] && ![variant_isset python27] && ![variant_isset python36] &&
+    ![variant_isset ruby] && ![variant_isset tcl] && ![variant_isset lua]} {
```

Comment:
This `if` block only needs to include the `pythonXY` variants. But it is alright as is.

Reply:
I don't think so, this only takes care of setting some default (in this case python37) if no variant is selected by the user. A user can still specify multiple variants if he/she wants to do and the "normal" variant conflicts will be honored as usual. Or am I missing your point here?

Reply:
You are missing the point. I did approve of this change. But it means that if the user selects, for example, the `+ruby` variant, then no python variant gets set. The `if` block is only needed when the user selects a non-default python variant because they all conflict with each other.

Reply:
Only the conflicting ports, i.e. the other python variants, need to be in this `if` block.

Reply:
Yes, it is intentional (at least for me) that the default (```python37```) variant is only selected when the user does not specify something else. There is no reason to assume that someone actually wants a/any Python binding when selecting another language.